client/.eslintrc.json

```diff
@@ -32,6 +33,7 @@
         "no-param-reassign": "off", // should refactor later
         "no-return-assign": "off", // should refactor later
         "spaced-comment": "off",
-        "arrow-body-style": "off" // should refactor later
+        "arrow-body-style": "off", // should refactor later
+        "linebreak-style": "off"
```

Comment:
If we're ignoring this rule, does this mean that when a Mac/Linux dev uses different line breaks, it registers as a diff, even though the code may not have changed?

Reply:
Saw this in a discussion from @cyr17 @vgoh1 in PR#198. So I wasn't sure if there's a better approach to surpressing linebreak style.

Reply:
I don't think it's being registered as diff? Because we have people using different OS and pushing to the repo and that hasn't been an issue. According to [eslint](https://eslint.org/docs/rules/linebreak-style): `Many versioning systems (like git and subversion) can automatically ensure the correct ending. However to cover all contingencies, you can activate this rule.` so I don't think we have to worry about this rule going in, since git should fix it for us.

Reply:
Makes sense, resolving.